hoot-core/src/main/cpp/hoot/core/io/OsmJsonReader.cpp

```diff
@@ -446,5 +442,120 @@ void OsmJsonReader::scrubBigInts(QString &jsonStr)
   jsonStr.replace(rx2, "\\1\"\\2\"\\3");
 }
 
+void OsmJsonReader::_readFromHttp()
+{
+  if (!_url.isValid())
+    throw HootException("Invalid URL: " + _url.toString());
+  //  Update the `srsname` parameter to use EPSG:4326
+  if (_url.hasQueryItem("srsname"))
+  {
+    _url.removeQueryItem("srsname");
+    _url.addQueryItem("srsname", "EPSG:4326");
+  }
+  bool split = false;
+  int numSplits = 1;
+  vector<thread> threads;
+  //  Check if there is a bounding box
+  if (_url.hasQueryItem("bbox") && _runParallel)
+  {
+    QStringList bbox = _url.allQueryItemValues("bbox");
+    //  Parse the bounding box
+    geos::geom::Envelope envelope = GeometryUtils::envelopeFromConfigString(bbox.last());
+    //  Check if the bounding box needs to be split
+    int lon_div = 1;
+    int lat_div = 1;
+    //  Don't split an envelope if it is just a little bigger than the prescribed max
+    if (envelope.getWidth() > _coordGridSize * 1.5)
+      lon_div = (int)std::ceil(envelope.getWidth() / _coordGridSize);
+    if (envelope.getHeight() > _coordGridSize * 1.5)
+      lat_div = (int)std::ceil(envelope.getHeight() / _coordGridSize);
```

Comment:
Get the number of divisions in each lat/lon direction if the original is larger than about 1/3 of a degree.  If it is slightly larger than the 0.25x0.25 degree box it will still be run as one request.  Larger than that, it will be split.